### ISSUE TITLE
webm drag to left to contract broken

### ISSUE THREAD

**Issue by fgsfdss** [ccd0/4chan-x#1857]
Stopped working sometime in the past week, presumably with the latest FF update.
FF 59 with Violentmonkey 2.8.30.
Pressing the 'contract' button still works.

**Comment by SchiraChan** [comment]
Still broken. FF 59.0.2 & Tampermonkey.
If it's a Firefox issue: Is there a fix?

**Comment by lc-guy** [comment]
still happening

**Comment by fgsfdss** [comment]
Still broken. Does it work on Chrome?

**Comment by SchiraChan** [comment]
Still broken. Firefox 61.0.1

**Comment by lakitu13** [comment]
Something to enable gestures again would be nice, but a simple button to contract the webm would be fine as well.

**Comment by fgsfdss** [comment]
You mean this?
![image](https://user-images.githubusercontent.com/5033279/45021916-22e42d80-b033-11e8-9fc5-089df9b4de6e.png)

**Comment by lakitu13** [comment]
Yes, I realized it was there after making the post but forgot to delete it.
